.github/pull_request_template.md

```diff
@@ -0,0 +1,6 @@
+<!--
+
+IMPORTANT: Please do not create a pull request which adds new functionality or modifies existing one without creating an issue first.
```

Comment:
It should be possible to just submit minor functionality. We should mention it's about major changes.